src/everest/simulator/simulator.py

```diff
@@ -132,7 +135,18 @@ def __call__(
                 case_data.append((real_id, controls))
 
         with open_storage(self._ert_config.ens_path, "w") as storage:
-            sim_context = self.start(f"batch_{self._batch}", case_data, storage)
+            if self._experiment_id is None:
+                experiment = storage.create_experiment(
+                    name=f"EnOpt@{datetime.now().strftime('%Y-%m-%d@%H:%M:%S')}",
+                    parameters=self.ert_config.ensemble_config.parameter_configuration,
+                    responses=self.ert_config.ensemble_config.response_configuration,
+                )
+
+                self._experiment_id = experiment.id
+
+            experiment = storage.get_experiment(self._experiment_id)
```

Comment:
Could be in an else clause, right? Any reason to pull the experiment again out of the storage if we just made it?

Reply:
Not really, putting it in an else clause